spec/models/generic_work_spec.rb

```diff
@@ -14,6 +14,16 @@
     it { is_expected.to eq 'curation_concerns_generic_work' }
   end
 
+  describe '#object_state' do
+    let(:work) { described_class.new }
+    let(:inactive) { ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive') }
```

Comment:
Not *necessarily* true. We've already got https://github.com/ruby-rdf/rdf-vocab/blob/develop/lib/rdf/vocab/fcrepo4.rb in there, and there are a handful of Fedora/Hydra folks shepherding that gem along.